config/localstack/3-setup-index.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo
+
+export AWS_ACCESS_KEY_ID=UNSET
+export AWS_SECRET_ACCESS_KEY=UNSET
+export AWS_DEFAULT_REGION=eu-west-2
+export ELASTIC_MAPPING_PATH=${ELASTIC_MAPPING_PATH:-"@elastic_mapping.json"}
+
+## make sure that localstack is running in the pipeline
+until curl http://localstack:4566/health --silent | grep -q "running"; do
+   sleep 5
+   echo "Waiting for LocalStack to be ready..."
+done
+
+until curl http://localstack:4571 --silent | grep -q "elasticsearch"; do
+   sleep 10
+   echo "Waiting for ES 6.7.0 to be ready..."
```

Comment:
```suggestion
   echo "Waiting for ElasticSearch to be ready..."
```
Maybe remove version number so when we update it we don't have to remember to change it here